tree/ntuple/v7/test/ntuple.cxx

```diff
@@ -815,3 +818,44 @@ TEST(RNTuple, ReadString)
    int nElementsPerPage = ntuple->GetDescriptor().GetClusterDescriptor(0).GetPageRange(1).fPageInfos.at(1).fNElements;
    EXPECT_EQ(contentString, viewSt(nElementsPerPage/7));
 }
+
+
+TEST(RNTuple, LargeFile)
+{
+   FileRaii fileGuard("test_large_file.root");
+
+   auto modelWrite = RNTupleModel::Create();
+   auto& wrEnergy  = *modelWrite->MakeField<double>("energy");
```

Comment:
Side note, it still read awkward (to me) to have to use a double dereference ('*' and '->') ...